bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/filter/RowFilters.java

```diff
@@ -0,0 +1,267 @@
+package com.google.cloud.bigtable.filter;
+
+import java.util.Arrays;
+import java.util.List;
+
+import com.google.api.client.util.Preconditions;
+import com.google.bigtable.v2.ColumnRange;
+import com.google.bigtable.v2.RowFilter;
+import com.google.bigtable.v2.RowFilter.Chain;
+import com.google.bigtable.v2.RowFilter.Condition;
+import com.google.bigtable.v2.RowFilter.Interleave;
+import com.google.bigtable.v2.TimestampRange;
+import com.google.protobuf.ByteString;
+
+public final class RowFilters {
+
+  public enum EdgeType {
+    None, Open, Closed;
+  }
+
+  public final static class ColumnRangeFilterBuilder {
+    private String family = null;
+    private ByteString start = null;
+    private ByteString end = null;
+    private EdgeType startEdgeType = EdgeType.None;
+    private EdgeType endEdgeType = EdgeType.None;
+    
+    public ColumnRangeFilterBuilder family(String family) {
+      this.family = family;
+      return this;
+    }
+
+    public ColumnRangeFilterBuilder startOpened(ByteString start) {
+      this.start = start;
+      this.startEdgeType = EdgeType.Open;
+      return this;
+    }
+
+    public ColumnRangeFilterBuilder startClosed(ByteString start) {
+      this.start = start;
+      this.startEdgeType = EdgeType.Closed;
+      return this;
+    }
+
+    public ColumnRangeFilterBuilder endOpened(ByteString end) {
+      this.end = end;
+      this.endEdgeType = EdgeType.Open;
+      return this;
+    }
+
+    public ColumnRangeFilterBuilder endClosed(ByteString end) {
+      this.end = end;
+      this.endEdgeType = EdgeType.Closed;
+      return this;
+    }
+
+    public RowFilter build() {
+      return R.columnRange(family, start, startEdgeType, end, endEdgeType);
+    }
+  }
+
+  public final static class R {
```

Comment:
I would recommend to make this a static instance instead of a static class so that users can alias it